update-version.sh

```diff
@@ -2,8 +2,11 @@
 
 set -e
 
-# find `ablyAssetTrackingLibraryVersion = "current_version"` and replace `current_version` with $1
-# script starts searching the pattern in project root directory recursively
+# This script is updating AAT Library version, it is looking for regex pattern `(ablyAssetTrackingLibraryVersion = ").+(")` and replace 
```

Comment:
I think this change probably belongs in #186, not here?

Reply:
yes, and my answer is here https://github.com/ably/ably-asset-tracking-swift/pull/186#issuecomment-976412144 :)

Reply:
Sorry, I'm not sure I understand. Since this is a PR on top of `feature/lib-versioning`, then I don't expect to see any changes related to introducing versioning (e.g. this script) in this PR's diff. Do you perhaps need to rebase on / merge in the latest version of `feature/lib-versioning`?

Reply:
I've reverted this commit, no changes visible in this PR